libs/MeshKernelApi/include/MeshKernelApi/MeshKernel.hpp

```diff
@@ -649,6 +649,16 @@ namespace meshkernelapi
         /// @returns Error code
         MKERNEL_API int mkernel_get_interpolation_type_float(int& type);
 
+        /// @brief Get the integer indicating the interpolation type double
```

Comment:
It is better to maintain enums in Python and .NET rather than bloat the API.

Reply:
Although having an API has its benefits. Changing the enum values as you indicated above would not result in additional pull requests in meshkernelNET and meshkernelPY.